app/src/main/java/com/andrew/tasky/presentation/adapters/AttendeeItemAdapter.kt

```diff
@@ -32,16 +36,15 @@ class AttendeeItemAdapter(
                 creatorTextView.isVisible = false
             }
 
-            if(attendees[position].contains(" ")){
-                val initials = attendees[position]
-                    .replace("^\\s*([a-zA-Z]).*\\s+([a-zA-Z])\\S+$"
-                        .toRegex(), "$1$2").uppercase()
+            //Adds full name to card
+            attendeeFullNameTextView.text = attendees[position].name
 
-                attendeeInitialsTextView.text = initials
-            }
-            else{
-                val initials = attendees[position].take(2).uppercase()
-                attendeeInitialsTextView.text = initials
+            //Adds initials to card
+            attendeeInitialsTextView.text = StringToInitials().convertStringToInitials(attendees[position].name)
+
+            //Adds on click listener to delete button
+            deleteAttendeeButton.setOnClickListener {
```

Comment:
Comments like these just mess up the codebase in the long run. My grandma could probably even tell you what these lines do 😁 
In general, I only comment on something if it's unclear WHY something happens. For example if you add a random `delay(10)` to your code because it might be necessary as a workaround for a bug, that's not clear to anyone. In that case, it totally makes sense to explain in a comment why that delay was added.

If you feel like you need a comment to explain WHAT happens because it's not clear otherwise, that's usually a sign that you can improve your code readability by adding a well named function, variable or class.